### PR TITLE
Spain: Update Capacities

### PR DESCRIPTION
**Pull request by nessie2013**
Updated capacities with REE, IRENA and ENTSO-E Data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -308,9 +308,10 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
...
-- Spain:[REE](https://www.ree.es/es/datos/publicaciones/series-estadisticas-nacionales)
-  - Hydro: [IRENA](http://resourceirena.irena.org/gateway/countrySearch/?countryCode=ESP)
-  - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show?name=&defaultValue=false&viewType=TABLE&areaType=BZN&atch=false&dateTime.dateTime=01.01.2017+00:00|UTC|YEAR&dateTime.endDateTime=01.01.2017+00:00|UTC|YEAR&area.values=CTY|10YES-REE------0!BZN|10YES-REE------0&productionType.values=B01&productionType.values=B02&productionType.values=B03&productionType.values=B04&productionType.values=B05&productionType.values=B06&productionType.values=B07&productionType.values=B08&productionType.values=B09&productionType.values=B10&productionType.values=B11&productionType.values=B12&productionType.values=B13&productionType.values=B14&productionType.values=B20&productionType.values=B15&productionType.values=B16&productionType.values=B17&productionType.values=B18&productionType.values=B19)
+- Spain:
+  - Other: [REE](https://www.ree.es/es/datos/publicaciones/series-estadisticas-nacionales)
```
Comment (lorrieq) [inline README.md:312]: Move Other to the bottom

**Comment by lorrieq** [comment]
Hey @nessie2013, thanks for the update and apologies for my delay in getting back.

My only remaining comment is around the big jump in gas capacity on Entsoe. I believe the previous value was also from Entsoe so I am surprised to see a 4GW increase. I wonder if we can cross check this somewhere to verify this makes sense as it is strange.

**Comment by nessie2013** [comment]
hmm looking on Entsoe, Gas capacity has been 30683 | 30266 | 30012 for 2018-2020, I can't find the original number source just looking quickly, it might of been from REE?

**Comment by lorrieq** [comment]
You're right, it's from REE. The row "Ciclo combinado" matches at 26,284. It's a little unclear what belongs to gas from REE. There are also rows named "Fuel + Gas" (2,447) and "Térmica no renovable/Cogeneración y resto/Cogeneración" (5,672). Summing the first two and ~25% of cogeneracion amounts to the Entsoe figure.

Given this I'm happy to go with Entsoe given the uncertainty around what is Gas in REE and considering the above combination from REE seems reasonable.

**Pull request merged**